package.json

```
@@ -40,7 +40,7 @@
         "theme": "dark"
     },
     "engines": {
-        "vscode": "^1.68.0"
+        "vscode": "^1.73.0"
```

```suggestion
        "vscode": "^1.73.0-insider"
```

I think this is what @DonJayamanne is suggesting.

---

@karrtikr was suggesting this is not needed since it is already a month since this API was out.

---

```suggestion
        "vscode": "^1.72.0"
```

Then I think that's what @karrtikr is suggesting.

---

I think bumping engine in consumer extensions should be enough.

Regardless, bumping our engine version only makes sure we don't install Python ext version until next version of VSCode is released, we would rather want the latest bits of Python are available on every version of VSCode. So I don't see the reasoning here.

---

provided the api and any bugs haven't been fixed in this area of the python extension then I'm happy for this not to change

---

@karrtikr I have tested the Python stable and pre-release and there's definitely a big change.
If you install stable version of Python extension, we don't get a list of the interpreters at all, the kernel picker doesn't get any environments.
If you install the pre-release extension of the Python extension, then things magically work.

I.e. theres definitely a behaviour change, hence this must be setup for things in Jupyter extension to work correctly.
Please feel free to validate this at your end, possible the refresh code in python extension has changed in the pre-release version

---

> we don't get a list of the interpreters at all

There's a slight behavior change, but nothing that should lead to this. We can just update vscode engine to be safe, although I don't think even that is needed due to https://github.com/microsoft/vscode-python/pull/20058#discussion_r1004863125.

@karthiknadig Please go ahead and update engine to insiders.